### PR TITLE
PR: Add a test to ensure all fonts used by QtAwesome have a unique family name

### PR DESCRIPTION
**Pull request by jnsebgosselin**
Add a regression test for Issue #107 that assert that each font used by qtawesome has a unique name.

### CODE REVIEW

**Comment by jnsebgosselin** [comment]
@ccordoba12 AppVeyor is failing as expected for the right reason.

How can I fix the import error in cicleci?

**Comment by jnsebgosselin** [comment]
@ccordoba12 I manage to get things to work. The test are failing as expected. Rebasing after PR #109 is merged should make them pass.

File: appveyor.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ install:
     - "powershell ci-helpers/appveyor/install-miniconda.ps1"
     - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
     - "activate test"
+    - "conda install -c conda-forge pytest-qt"
```
Comment (ccordoba12) [inline appveyor.yml]: Please remove this line and instead add below `CONDA_DEPENDENCIES` this line

`PIP_DEPENDENCIES: "pytest-qt"`

**Comment by jnsebgosselin** [comment]
tyvm for the review @ccordoba12

**Pull request merged**